### DISCUSSION TITLE
Found guides / tutorials that includes the usage of this repo

### DISCUSSION THREAD

**Discussion by Purfview** [Purfview/whisper-standalone-win#315] [Show and tell]
"Show and tell" is empty, let's start with this thread. Post a link if you found/created a guide/tutorial ect.

Don't take "Amateur hour" links too seriously, some thing said there are wrong, or shows how to do things in a silly way or it's plainly showcases for "what not to do" , anyway, newbies may find there something useful for them.

## Recommended to watch:
[Convert Any Video or Audio to Text for Free | No Limits!](https://www.youtube.com/watch?v=dJ6HUzxPWHg)

## Amateur hour, beginners for the beginners:
[[TUT] whisper-standalone-win - Einfach Untertitel für Videos generieren lassen [4K | DE]](https://www.youtube.com/watch?v=XtcSqvx2Yfo)
[Speech to Text in Windows using Whisper-Faster-XXL and Right-Click Context Menu](https://www.youtube.com/watch?v=XtcSqvx2Yfo)
[Comparativa Whisper en Subtitle Edit: OpenAI vs CPP vs Const-me vs Ctranslate2 vs Faster Whisper](https://www.youtube.com/watch?v=VgdEG_kb6Bk)
[Faster Whisper OpenAi - Instrukcja instalacji, konfiguracji i prostej transkrypcji](https://www.youtube.com/watch?v=rPkbzM6mqrw)
[Automata feliratkészítés minden eddiginél gyorsabban a saját számítógépünkön teljesen ingyenesen](https://www.youtube.com/watch?v=fG56FlIz85s)
[Sbobinare Audio e Video in Testo GRATIS e Facile!](https://www.youtube.com/watch?v=5OiW1m056cc)
[Installare CUDA e cuDNN per le trascrizioni con Whisper](https://www.youtube.com/watch?v=qZckN7D09wE)
[Cómo convertir Audio a Texto en tu propia PC. (Incluye traducción de audios)](https://www.youtube.com/watch?v=OfkCC6eGIO0)

## Expert/proficient guides/tutorials/videos:
[Convert Any Video or Audio to Text for Free | No Limits!](https://www.youtube.com/watch?v=dJ6HUzxPWHg)
[Spracherkennung (fast) perfekt und in 3 Minuten installiert: Whisper Standalone für Windows](https://www.youtube.com/watch?v=20zr9MgI0LE)
[Subtitle Edit 4.0.5 - What's New](https://www.youtube.com/watch?v=zenmzgJ2Fxw)
["äh"-Füllwörter-Erkennung für Audacity und Davinci Resolve mit Whisper und GPT-4 (Tutorial)](https://www.youtube.com/watch?v=3NNTWEM7yVA)
[AviUtl + 拡張編集 + Whisper で音声から自動で文字起こしができるプラグイン Subtitler](https://www.nicovideo.jp/watch/sm44079956)
[Subtitle Edit installation common problem solving guide](https://www.youtube.com/watch?v=m7YTcvJLAOg)
[Faster-Whisper-XXL 명령어 옵션 총정리: 한국어 완역 및 전문가 해설, 명령어, 설명, 파라미터](https://www.wonsiin.com/2025/01/faster-whisper-xxl.html#)

## Other/Random things:
[Media transcription Faster Whisper XXX thoughts 241120](https://www.youtube.com/watch?v=N8JwbmFY_zE) 😁

**Comment by chaoticbovine** [comment]
Found the following: [Portable Whisper Speech to Text with Speaker Diarization and VAD | Purfview Faster Whisper XXL](https://www.youtube.com/watch?v=S6YFeKJIURk)
